### ISSUE TITLE
Help me with integration "react-native-google-fit" into my project

### ISSUE THREAD

**Issue by liudmyla-schastlyvets** [StasDoskalenko/react-native-google-fit#105]
Can anybody help me what may be the reason why Google Fit is not available for the current account (`isAvailbale` method returns `false`) and authorization is canceled on `authorize` method?
It was Expo project which was ejected to ExpoKit. 
I followed all the installation instructions provided in the react-native-google-fit.

**Comment by ensargunesdogdu** [comment]
+1

**Comment by antoinedelp** [comment]
In my case, the issue was an incorrect initialization of Google Developper API Key. Double check that the key is correct. 
https://developers.google.com/fit/android/get-api-key
Do you have a different configuration for debug and release?

**Comment by liudmyla-schastlyvets** [comment]
@antoinedelp Yes, I have different configurations for debug and release.
After creating a separate application in google console and adding OAuth 2.0 client IDs application could authorize into GoogleFit. It does not work for my main google project.
Also my project is integrated with firebase. For the android application fingerprints also were added in firebase console. I used `google-services.json` and notifications work fine.
